jobs/ci-kubernetes-e2e-gke-gpu-1-7.env

```diff
@@ -0,0 +1 @@
+PROJECT=k8s-jkns-e2e-gke-gpus-1-7
```

Comment:
now we uses `--gce-project=k8s-jkns-e2e-gke-gpus-1-7` in config.json (ref https://github.com/kubernetes/test-infra/pull/3528)